JamesB.txt

```diff
@@ -0,0 +1,5 @@
+Hello my name is James.
+
+Three words that describe me are personable, easy-going, and witty.
+
+Dogs are my favorite animal. Between my parents and my brothers, my family has six dogs. I adopted my dog from the Bronx when I was in college. Her name is Daisy and she is a red-nose pitbull. She is turning 6 years old in June. 
```

Comment:
That's awesome to hear man. I'm a dog-person too.  Good job on the assignment. Looks like you have everything down.